src/lib/y2storage/autoinst_profile/drive_section.rb

```diff
@@ -619,7 +619,7 @@ def use_value_from_string(use)
       #   nil if the root filesystem is not applicable.
       def enabled_snapshots?(filesystems)
         root_fs = filesystems.find(&:root?)
-        return nil if root_fs.nil? || (root_fs.multidevice? && !btrfs_drive_section?)
+        return false if root_fs.nil? || (root_fs.multidevice? && !btrfs_drive_section?)
```

Comment:
and this cause several test failures. I am not sure if it is correct change. On one hand it is `?` method, but nil is documented there. Also not sure if it is common in storage or one single occurrence.